.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,52 @@
+name: test-integration
+on:
+  pull_request:
+  push:
+#    branches: [ master, develop ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ runner.os }}-composer-
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Generate SSL key and certificate
+        run: |
+          sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ./ci/files/tiqr.example.com.key -out ./ci/files/tiqr.example.com.crt -subj "/C=NL/ST=Netherlands/L=Amsterdam/O=TEST/CN=tiqr.example.com"
+      - name: Init environment
+        run: |
+          cd ci/docker
+          docker-compose up -d
+      - name: Install dependencies
+        run: |
+          cd ci/docker && docker-compose exec -T tiqr.php.test.vm.openconext.org bash -lc '
+          composer install
+          yarn
+          cp .env.ci .env
+          cp config/legacy/parameters.yaml.dist config/legacy/parameters.yaml
+          '
+      - name: Build frontend assets
+        run: cd ci/docker && docker-compose exec -T tiqr.php.test.vm.openconext.org bash -lc 'yarn encore production'
+      - name: Run test scripts
+        run: cd ci/docker && docker-compose exec -T tiqr.php.test.vm.openconext.org bash -lc ' composer test '
+      - name: Output log files on failure
+        if: failure()
+        run: cd ci/docker && docker-compose exec -T tiqr.php.test.vm.openconext.org -c 'cat var/log/webtest.log'
```

Comment:
I don't think the webtest file will exist. See feedback on the previous PR's where we had similar logging challenges. And make this line uniform with the other.

Reply:
Resolved it the same way as [Stepup-Middleware](https://github.com/OpenConext/Stepup-Middleware)